src/Verification/Shipment/CanSendShipment.php

```diff
@@ -81,11 +81,11 @@ public function verify(Order $order, OrderState $orderState)
         }
 
         if (!$this->hasPaymentInformation($order->id)) {
-            throw new ShipmentCannotBeSentException('Shipment information cannot be sent. Order has no payment information', ShipmentCannotBeSentException::ORDER_HAS_NO_PAYMENT_INFORMATION, $order->reference);
+            return false;
```

Comment:
Why would these exceptions be removed while AUTOMATIC_SHIPMENT_SENDER_IS_NOT_AVAILABLE is left?

Reply:
shipmentSenderHandler is called on order status change as well. When order is created and initial order state is created, it is possible to ship order on this initial status. But payment is not yet inserted into database so removed exceptions would be thrown even when there is no actual problem. 
About left exception: I don't see why we need to leave it so I will remove it as well.